CyberSourceSamples/src/xml/xmlSamples/androidPayAuth.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<requestMessage
+    xmlns="urn:schemas-cybersource-com:transaction-data-1.129">
+    <merchantID >venky_test</merchantID>
```

Comment:
In all the csv and xml payload files, keep it same. Either "your_merchant_id" or "merchant_id".